keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/ResourceEvent.kt

```diff
@@ -296,7 +296,8 @@ data class ResourceCheckError(
   override val name: String,
   override val application: String,
   override val timestamp: Instant,
-  val exception: Throwable
+  val exceptionType: Class<Throwable>,
+  val exceptionMessage: String?
```

Comment:
Semi-related change. I noticed the resource history has big blobs of stack trace when there are resource check errors. Seems undesirable to store that. I'll need to flush DB before rolling this out.